### PR TITLE
Feat: Create a summary page to aggregate test reports (#4)

### PR DESCRIPTION
**Pull request by tedd-E**
This PR addresses issue #4.

Aggregate test reports are accessible under the `Summaries` page, with an overview of the tests run in each project.
Each square represents a test suite. A square is green if no specs are skipped or failed, yellow if no specs are failed but some specs are skipped, and red otherwise.

**Note:** at the moment, clicking on any of the squares will redirect back to the `Test Runs` page. Once test run filtering has been introduced, the redirect can be changed to show the corresponding test suite for that specific square.

image

### CODE REVIEW

**Comment by anoop2811** [comment]
Hey @tedd-E : The page looks great! How is this coming along?

**Comment by tedd-E** [comment]
Hi @anoop2811, the majority of work is finished, there are some minor details left that I'm planning on changing. Feel free to let me know if you see any areas for improvement for the existing code in the meantime, I'll move the PR out of the draft state once everything is wrapped up.

**Comment by anoop2811** [comment]
@tedd-E , would be great to deploy this in our internal env as it would help in the reviewing whenever you are ready

**Comment by tedd-E** [comment]
@anoop2811 I've deployed the changes to the `dev` environment, link is in contributor's channel if needed
image

**Review by gowtham-ra — APPROVED** [review]

File: src/providers/testrun-provider.ts [hunk 1]
```diff
@@ -5,9 +5,10 @@ if (!import.meta.env.VITE_FERN_REPORTER_BASE_URL) {
     console.log('Error: FERN_REPORTER_BASE_URL is not set');
 }
 
-const API_URL = import.meta.env.VITE_FERN_REPORTER_BASE_URL;
+export const API_URL = import.meta.env.VITE_FERN_REPORTER_BASE_URL;
+console.log("API_URL", API_URL)
```
Comment (gowtham-ra) [inline src/providers/testrun-provider.ts:9]: nit: this log can be removed

File: src/pages/test-summaries/summary-utils.tsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+                                        <div><b>{item.TotalSkippedSpecRuns}</b> specs skipped</div>
+                                    )}
+                                    <div> {new Date(item.StartTime).toLocaleString()} </div>
+                                </div>
+                            </>
+                        }
+                        key={item.SuiteRunID}
+                    >
+                        <div style={getBoxStyle(item)}
+                             onClick={() => window.open('/testruns/', '_blank')}>
```
Comment (gowtham-ra) [inline src/pages/test-summaries/summary-utils.tsx:68]: small improvement for the future: When we click a particular test run, we can open the expandable row header for that particular test run. Now it re-directs to the whole page and it's on the user to find the test run.

File: src/providers/testrun-provider.ts (hunk 1 above)
Comment (tedd-E) [reply src/providers/testrun-provider.ts]: Removed the log

File: src/pages/test-summaries/summary-utils.tsx (hunk 2 above)
Comment (tedd-E) [reply src/pages/test-summaries/summary-utils.tsx:68]: Good idea, I'll create a new issue with a list of enhancements for the summary page

**Pull request merged**